README.md

```diff
@@ -89,8 +84,9 @@ High level stuff planned for Q4 2017 (October - December):
 | Feature          | Status                              | Docs         | Issue          |
 |------------------|-------------------------------------|--------------|----------------|
 | tree             |      In-progress ([sneak peek][31]) |           -  |  [#3175][3175] |
-| sticky-header    |        In-progress, planned Q3 2017 |           -  |   [#474][0474] |
-| virtual-repeat   |        Not started, planned Q4 2017 |           -  |   [#823][0823] |
+| sticky-header    |        In-progress, planned Q2 2018 |           -  |   [#474][0474] |
+| virtual-repeat   |        Prototyping, planned Q1 2018 |           -  |   [#823][0823] |
+| badge            |        In-progress, planned Q4 2017 |           -  |   [#823][0823] |
```

Comment:
I guess you forgot to edit the linked issue, https://github.com/angular/material2/issues/3058 is the right one.

Reply:
Good catch, fixed